tools/ds-identify

```diff
@@ -475,7 +483,7 @@ detect_virt() {
             case "$out" in
                 hv) virt="microsoft" ;;
                 vbox) virt="oracle" ;;
-                generic) "vm-other";;
+                generic) virt="vm-other";;
```

Comment:
Was this a bug we just happened to never run into?

Reply:
Yes